docs/about/overview.md

```diff
@@ -52,59 +72,129 @@ end
 
 ---
 
-## What You Can Do with the NeMo Guardrails Library
+## Use Cases
 
 The following are the top use cases of the NeMo Guardrails library that you can apply to protect your LLM applications.
 
-::::{grid} 1 1 2 2
-:gutter: 3
-:class-container: sd-equal-height
+:::{dropdown} 🛡️ Add Content Safety
 
-:::{grid-item-card} Content Safety - Text
-:link: ../getting-started/tutorials/nemotron-safety-guard-deployment
-:link-type: doc
+Content safety guardrails help ensure that both user inputs and LLM outputs are safe and appropriate.
+The NeMo Guardrails library provides multiple approaches to content safety:
 
-Deploy Nemotron Content Safety to detect harmful content in text inputs and outputs.
-:::
+- **LLM self-checking**: Use the LLM itself to check inputs and outputs for harmful content.
+- **NVIDIA safety models**: Integrate with [Llama 3.1 NemoGuard 8B Content Safety](https://build.nvidia.com/nvidia/llama-3_1-nemoguard-8b-content-safety) for robust content moderation.
```

Comment:
This should reference the newer Nemotron Safety Guard v3 model
https://build.nvidia.com/nvidia/llama-3_1-nemotron-safety-guard-8b-v3